src/harness/unittests/programMissingFiles.ts

```diff
@@ -48,26 +48,26 @@ namespace ts {
             const program = createProgram(["./nonexistent.ts"], options, testCompilerHost);
             const missing = program.getMissingFilePaths();
             assert.isDefined(missing);
-            assert.deepEqual(missing, ["d:/pretend/nonexistent.ts"]); // Absolute path
+            assert.deepEqual(missing, ["d:/pretend/nonexistent.ts" as Path]); // Absolute path

```

Comment:
What makes this cast (and the similar ones below) required now, but not before? Should it have been required before?